pyproject.toml

```diff
@@ -12,13 +12,19 @@ scipy = "^1.7.3"
 matplotlib = "^3.5.1"
 pyampd = "^0.0.1"
 
+
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.5"
 ipykernel = "^6.6.0"
 statsmodels = "^0.13.1"
 mypy = "^0.971"
 black = "^22.6.0"
 pytest-cov = "^3.0.0"
+pre-commit = "^2.20.0"
+isort = "^5.10.1"
+
+[tool.black]
+line-length = 79
```

Comment:
I've just added this so that we get a consistent line length in the pre-commit hooks and the github actions